chain/jsonrpc/client/src/lib.rs

```diff
@@ -92,7 +93,11 @@ where
             response.body().map(|body| match body {
                 Ok(bytes) => String::from_utf8(bytes.to_vec())
                     .map_err(|err| format!("Error {:?} in {:?}", err, bytes))
-                    .and_then(|s| serde_json::from_str(&s).map_err(|err| err.to_string())),
+                    .and_then(|s| {
+                        serde_json::from_str::<DeserializedWithDebug<_>>(&s)
+                            .map(|v| v.0)
+                            .map_err(|err| err.to_string())
+                    }),
```

Comment:
So this is data that comes from outside so we probably don’t actually care. We need the warnings when reading local files or data from the database.

Reply:
@mina86 could you clarify a bit about

> data from the database.

I am confused here -- data in the db is borsh-encoded, it's not self-describing, so we can't apply this to db data?

Reply:
Well, actually, VERSION_KEY is in JSON. ;) But yes, you’re right in general. I didn’t have anything specific in mind here. Just wanted to point out that we don’t want to warn on data controlled by third parties. Only on data that comes from us.